views/components/Graphs/GraphTable.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import {
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow,
+  Table,
+  Paper,
+} from '@material-ui/core'
+
+const GraphTable = ({ graph }) => {
+  if (!graph) return null
+
+  const [totalsSite, ...sites] = graph.dataBySite
+  return (
+    <Paper>
+      <Table>
+        <TableHead>
+          <TableRow>
+            <TableCell align="center">Site</TableCell>
+            {graph.labels?.map(({ name }) => (
```

Comment:
when would we have a graph and no labels?

Reply:
haha, sorry this is me trying to control something that might be a risk.
https://github.com/AMP-SCZ/dpdash/pull/175/commits/6002f254766460fc968874ee0aef199e12df4bf6